sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -132,7 +132,7 @@ async def read(self, **kwargs: Any) -> Dict[str, Any]:
         :rtype: Dict[str, Any]
         """
         # TODO this helper function should be extracted from CosmosClient
```

Comment:
Think this TODO can be deleted now?

Reply:
We're still using the CosmosClient method in the line right below (137) so not yet unfortunately.